Website/app/docs/Quickstart.md

```diff
@@ -0,0 +1,19 @@
+# A general overview of Dirfty
+
+Dirfty is an open source Interactive file downloader software built using Java and NextJS, Its backend is powered by Java and the front end consists of NextJS and TailwindCSS
+
+With the help of Dirfty, you can download files and videos from various sources such as YouTube, Instagram etc.. You can also download files by providing the link address if it's downloading way too slowly inside your browser.
+
+## Main Features of Drifty
+
+- Free and Open Source forever
+- Faster downloading of files, just like IDM 
+- Youtube, Instagram, Twitter/X etc.. Platforms supported
+- Both CLI (Command Line Interface) and GUI (Graphical User Interface) are available
+- Parallel downloading support provided
+
+# Let's get you running
+
+##  A simple demo on how to get started with the GUI of Drifty
+
+https://github.com/hasnainmakada-99/Drifty/assets/82728823/980cafd3-6e3e-4b56-a890-b96ec47b7593
```

Comment:
@hasnainmakada-99 I think you can remove the sound and add texts in the video to show what you are pointing to in the video. You can remove redundant stops like the user is waiting on the homepage of the Drifty Website for more than 25 seconds. Generally, the audio is muted in videos. So, it's better to include your words in the form of texts in the video. 
Two benefits are there :point_down: 
- Improving accessibility
- Reducing size of the video a bit

Reply:
Yeah that's true @SaptarshiSarkar12 what you are saying, But I am really bad with editing videos, so can't help you out on this one

Reply:
Okay. Will try to do that myself if possible. I'm also not too good at editing :upside_down_face:.

Reply:
Done :ballot_box_with_check:!
@hasnainmakada-99 You can see it below. Suggestions are welcomed :smiley:.

https://github.com/SaptarshiSarkar12/Drifty/assets/105960032/5bb21bf6-fb2b-4374-8d0b-14cac3c06908

Reply:
Looks great @SaptarshiSarkar12 , I think that we should include this inside the main deployment

Reply:
Yeah, can you please include that change in your next push?